### ISSUE TITLE
[PL][Spell][Mage] Living Bomb / Flame Orb

### ISSUE THREAD

**Issue by adryjan** [Atlantiss/BugTracker#3653]
Crit z eksplozji Living Bomb oraz Flame Orb powinien nakładać na cele Ignite. Nie chodzi mi o Doty tylko o direct dmg spowodowany wybuchem. Podrzucam filmik, 35/36 sekunda wybuch Living Bomb = Ignite: https://youtu.be/jh2VwFZcLaQ?t=35s .

Linki:
Living Bomb - http://wow.gamepedia.com/Living_Bomb
Flame Orb - http://wow.gamepedia.com/Flame_Orb
Ignite - http://wow.gamepedia.com/Ignite

Dodatkowo komentarz na Tauri - Veins:
''Prior to patch 4.1, ticks of Living Bomb Living Bomb and Flame Orb Flame Orb could trigger Ignite. Now, they do not trigger Ignite anymore, which greatly limits the circumstances under which Ignite Munching can happen. It makes it much easier for Fire Mages to cast Combustion Combustion following a high critical hit. Indeed, Ignite Munching will now mostly occur when Flame orb and Living Bomb explode. As these are signaled by visual effects, you should be able to avoid using Combustion immediately after seeing these effects.'' 

http://tauri-veins.tk/fire-mage-wow-pve-dps-rotation-cooldowns-abilities

**Comment by sunshineinabox** [comment]
Ignite munching shouldn't be occurring at all, but they were happy to implement it hopefully they are as eager to implement this fix. Which is a legitimate fix.

**Comment by adryjan** [comment]
Dodaje SS, jeżeli są wątpliwości. LB crit x11, Ignite x0 (2x z Flamestrike który wrzucił mi combat na 3 targety i umożliwił dodowanie). 

http://i62.tinypic.com/s1stmu.jpg

**Comment by Jenovas** [comment]
Living Bombs explosion applies Ignite if it crits, which is wrong.

proof:
http://www.mmo-champion.com/threads/1036699-Flame-Orb-Ignite?p=14516327&viewfull=1#post14516327
"Flame Orb's explosion can not cause ignite, nor can living bomb's explosion cause it. I recently (and just now again) tested this, even SimulationCraft had this wrong until recently."

http://www.mmo-champion.com/threads/1038982-Ignite-munching-4-3?p=14575348&viewfull=1#post14575348
"Chiaroscuro, I spent some time on some target dummies the other day putting up LB x 3 and using Flame orb on CD for 5 minutes. Not once did ignite show up in my damage logs. This includes many living bomb explosions, it is a little harder to tell if I got a flame orb explosion, but I would say we can probably assume it behaves the same."
